connector_acp/data/document_data_view.xml

```diff
@@ -0,0 +1,13 @@
+<odoo>
+
+    <record id="etd_account_invoice" model="etd.document">
+        <field name="name">Invoice</field>
+        <field name="model">account.invoice</field>
+    </record>
+
+    <record id="etd_account_ticket" model="etd.document">
+        <field name="name">Ticket</field>
+        <field name="model">account.invoice</field>
+    </record>
+
+</odoo>
```

Comment:
This is a new data file, it needs to be added to the manifest.
Please add it in the `demo` key.